projects/packages/search/src/dashboard/components/record-meter/lib/record-info.js

```diff
@@ -52,7 +52,7 @@ export default function getRecordInfo( postCount, postTypeBreakdown, lastIndexed
 			postTypes
 		);
 
-		for ( let i = 0; i < numItems - 1; i++ ) {
+		for ( let i = 0; i < numItems; i++ ) {
```

Comment:
`addLabelsToPostTypeBreakdown` now hands us back a zero-indexed array, so we don't need the `-1`. This was resulting in the final post type never being displayed.